### PR TITLE
Breathing timer

### PR DESCRIPTION
**Pull request by sophieheb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
     "test:cov": "NODE_ENV=test jest --collect-coverage --watchAll"
   },
   "engines": {
-    "node": "12.x",
```
Comment (jits) [inline package.json:25]: Still need this for Heroku (I'm worried if we don't have this here we'll mistakenly use a newer version like v14, which could come with potential issues).

File: src/components/Header.js [hunk 2]
```diff
@@ -57,14 +103,75 @@ const Header = ({ menuContainer }) => {
...
       <Box p={2}>
-        <IconButton onClick={() => { setDrawerOpen(true); }}>
-          <NaturePeople />
+        <IconButton onClick={() => { setModalOpen(true); }}>
+          <SvgIcon component={BreathIcon} />
         </IconButton>
 
       </Box>
 
+      <Dialog
```
Comment (jits) [inline src/components/Header.js:113]: Just for future maintenance… could you move this to a separate component that's self contained?

**Review by jits — APPROVED** [review]
Couple of comments, otherwise LG!

File: package.json (hunk 1 above)
Comment (sophieheb) [reply package.json:25]: Added back in

File: src/components/Header.js (hunk 2 above)
Comment (sophieheb) [reply src/components/Header.js]: Done

**Pull request merged**